mute_spotify_timer.sh

```diff
@@ -3,16 +3,16 @@
 while true; do
 	# get Spotify playing song name
 	name=`dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.Get string:org.mpris.MediaPlayer2.Player string:Metadata | sed -n '/title/{n;p}' | cut -d '"' -f 2`
-
+	url=`dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.freedesktop.DBus.Properties.Get string:org.mpris.MediaPlayer2.Player string:Metadata | sed -n '/url/{n;p}' | cut -d '"' -f 2`
 	echo $name
-
-	if [[ "$name" = *"Advertisement"* || "$name" = *"Spotify"* ]]; then
+	
+	if [[ "$name" = *"Advertisement"* || "$name" = *"Spotify"* ]] || [[ $url == *"/ad/"* ]]; then
 		echo "Muting"
-		~/spotify-muter-linux/mute_app.sh spotify mute
-		sleep 25
+		~/git/spotify-muter-linux/mute_app.sh spotify mute
+		sleep 5
 	else
 		echo "Unmuting"
-		~/spotify-muter-linux/mute_app.sh spotify unmute
+		~/git/spotify-muter-linux/mute_app.sh spotify unmute
```

Comment:
Please remove the "git/" from the path so that a simple git clone in ~ works fine. Also at line 11.